instrumentation/grpc-1.6/javaagent/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ muzzle {
     module.set("grpc-core")
     versions.set("[1.6.0,)")
     assertInverse.set(true)
+    extraDependency("io.grpc:grpc-protobuf:1.6.0")
```

Comment:
`grpc-protobuf` added `io.grpc:grpc-core:1.6.0` to classpath, it causes muzzle-check check failed. Refer to https://scans.gradle.com/s/c6dqpidygm5gg/failure#1
You can try to use lower version dependency here, such as `io.grpc:grpc-protobuf:1.5.0` and check whether it can meet your needing.

Reply:
Thanks, I'll try it.